cluster/gce/gci/configure-helper.sh

```diff
@@ -1513,6 +1513,11 @@ function start-kube-apiserver {
     params+=" --experimental-encryption-provider-config=${encryption_provider_config_path}"
   fi
 
+  # disable using HPA metrics REST clients if metrics-server isn't enabled
```

Comment:
Does it work?

It seems the "horizontal-pod-autoscaler-use-rest-clients" is the controller manager flag, but here we are generating flags to apiserver...

Reply:
Thanks for catching this!
@kawych please fix

Reply:
Thanks, nice catch. Fixed & tested